locales/ar.yml

```diff
@@ -0,0 +1,11 @@
+---
+ar:
+  header:
+    title: "مدوّنة برلين للسلوك"
+    sub-title: "مدوّنة سلوك لكل المجموعات و الفعاليات"
+  supporters:
+    title: "المتبنّون"
+    sub-title: "المجموعات و الفعاليات التالية تتبنّى مدونة برلين للسلوك:"
+    add: "لإدراج مجموعة أو فعالية هنا أضفها في طلب سحب في %{link:our repository}."
+  footer:
+    attribution: "مدوّنة السلوك هذه مقتبسة من %{link:pdx.rb code of conduct}."
```

Comment:
I would simplify to:

`هذه المدونة مقتبسة من`

Reply:
@agharbeia @amrabdelwahab please note the you can also translate the link text: `%{link:*********}`. Feel free to translate “our repository” and “pdx.rb code of conduct” and check if the result looks fine in Middleman.

Reply:
@amrabdelwahab  I'd rather maintain the full name "مدونة السلوك" whenever possible. It's clearer this was, and usually it is done so in the more formal and semi-legal tone.

@hagenburger Thanks for the tip. "our repository" did slip from me. While I had chosen to leave "code of conduct" as is in order to imply that it is in English, and save the reader who is not interested the effort, and because I was thinking it's its title, and sense no Arabic edition exists of it then let's not make one up. But I'd like to know what the others think about this

Reply:
@agharbeia as this can be considered as as “brand name” you can leave it in English. If this mix of Latin and Arabic looks good enough, it’s OK :)